main.lua

```diff
@@ -56,6 +56,7 @@ SPEED = {
 }
 
 TEXT_BUFFER = {}
+-- CURSOR_POS_BUFFER = { x = 1, y = 1 }
```

Comment:
czy to dalej potrzebne?

Reply:
Prawdopodobnie będzie potrzebne w bliskiej przyszłości, dlatego aktualnie to zakomentowałem

Reply:
ok